src/Rx/Weno/wenoPharmaciesImport.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Rx\Weno;
+
+use Pharmacy;
+
+class wenoPharmaciesImport
+{
+    private $filename;
+    private $state;
+
+    public function __construct()
+    {
+        $this->filename = "../../../contrib/weno/WenoPharmacyDirectory2020-12-13.csv";
```

Comment:
take a path from the globals settings to avoid breaking this if this class gets moved around in the future

Reply:
done